app/controllers/tags_controller.rb

```diff
@@ -10,4 +12,27 @@ def index
       format.json { render :json => @tags }
     end
   end
+
+  def new
+    @tag = Tag.new
+    @title = "Create Tag"
+  end
+
+  def create
+    tag = Tag.create!(params.require(:tag).permit(:tag, :description, :privileged, :is_media, :inactive, :hotness_mod))
```

Comment:
Please break out the params filtering to a method for this and `update` to share.